paper/paper.md

```diff
@@ -5,44 +5,52 @@ tags:
   index: 3
-- Sorbonne Université, Ecole Normale Supérieure, PSL Research University, CNRS, Laboratoire des Biomolécules (LBM), F-75005 Paris, France
+- name: Mitochondria, Metals and Oxidative Stress group, Institut Jacques Monod, UMR 7592, Université de Paris, CNRS, F-75013 Paris, France.
   index: 4
-- Université de Paris, UFR Sciences du Vivant, F-75013 Paris, France
+- name: Sorbonne Université, Ecole Normale Supérieure, PSL Research University, CNRS, Laboratoire des Biomolécules (LBM), F-75005 Paris, France
   index: 5
+- name: Université de Paris, UFR Sciences du Vivant, F-75013 Paris, France
+  index: 6
 date: 27 May 2021
 bibliography: paper.bib
 ---
 
-# Statement of need
+# Background
+
+Molecular dynamics (MD) simulations of lipids are widely used to understand the complex structure and dynamics of biological or model membranes [@Tieleman1997; @Feller2000; @Lyubartsev2011]. They are very complementary to biophysical experiments and have thus become important to get insights at the microscopic scale. Many of them are performed using all-atom (AA) or united-atom (UA) representations. In AA force fields (such as CHARMM36 [@Klauda2010]), all the atoms are considered whereas in UA force fields (such as Berger [@Berger1997]) the aliphatic hydrogen atoms (Hs) are merged to their parent carbon into a larger particule representing a CH, CH2 or CH3 (e.g. a methyl group is represented by a single CH3 particle). In simulations of phospholipids, the use of UA representations allows to divide by almost 3 the number of atoms to simulate due to aliphatic chains of the lipids and reduce the computation time.
 
-Molecular dynamics (MD) simulations of lipids are widely used to understand the complex structure and dynamics of biological or model membranes [Tieleman1997; @Feller2000; @Lyubartsev2011]. They are very complementary to biophysical experiments and have thus become important to get insights at the microscopic scale. Many of them are performed using all-atom (AA) or united-atom (UA) representations. In AA force fields (such as CHARMM36 [@Klauda2010]), all the atoms are considered, whereas in UA force fields (such as Berger [@Berger1997]) the aliphatic hydrogen atoms (Hs) are merged to their parent carbon into a larger particule representing a CH, CH2 or CH3 (e.g. a methyl group is represented by a single CH3 particle). In simulations of phospholipids, the use of UA representations allows to divide by almost 3 the number of atoms to simulate because these molecules contain many aliphatic Hs.
 MD simulations of lipids are usually validated against experimental data [@Klauda2010] or used to help interpret experiments [@Feller2007]. One type of experiment which is widely for that is $^2H$ NMR. In this type of experiment, aliphatic Hs atoms are replaced by deuterons. $^2H$ NMR allows one to measure the order parameter of a given C-H bond (where the H is replaced by a deuteron):
```

Comment:
`One type of experiment which is widely for that` -->  `One type of experiment which is often used for that`